pkg/ccl/backupccl/backup_test.go

```diff
@@ -372,6 +372,51 @@ func backupAndRestore(
 		}
 	}
 }
+
+func TestBackupRestoreSystemTables(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+
+	const numAccounts = 0
+	_, _, sqlDB, _, cleanupFn := backupRestoreTestSetup(t, multiNode, numAccounts, initNone)
+	defer cleanupFn()
+
+	tables := []string{"descriptor", "users", "zones"}
+
+	sqlDB.Exec(t, `CREATE DATABASE system_new`)
+
+	expectedFingerprints := make(map[string][][]string)
+	for _, table := range tables {
+		expectedFingerprints[table] = sqlDB.QueryStr(t,
+			fmt.Sprintf("SHOW EXPERIMENTAL_FINGERPRINTS FROM TABLE system.%s", table),
+		)
+	}
+
+	sqlDB.Exec(t, `BACKUP DATABASE system TO $1`, localFoo)
+	sqlDB.Exec(t, `RESTORE system.* FROM $1 WITH OPTIONS ('into_db'='system_new')`, localFoo)
```

Comment:
any reason for the old options syntax?

Reply:
oh, because this was a couple weeks before that merged.

Reply:
Fixed.